server.js

```diff
@@ -1229,6 +1229,23 @@ module.exports.initExpress = function () {
       require('./pages/instructorQuestionManualGrading/instructorQuestionManualGrading'),
     ]
   );
+
+  app.use(
+    '/pl/course_instance/:course_instance_id/instructor/instance_question/:instance_question_id/clientFilesQuestion',
+    [
+      require('./middlewares/selectAndAuthzInstanceQuestion'),
+      require('./pages/clientFilesQuestion/clientFilesQuestion'),
+    ]
+  );
+
+  app.use(
+    '/pl/course_instance/:course_instance_id/instructor/instance_question/:instance_question_id/generatedFilesQuestion',
+    [
+      require('./middlewares/selectAndAuthzInstanceQuestion'),
+      require('./pages/studentGeneratedFilesQuestion/studentGeneratedFilesQuestion'),
```

Comment:
I did a double-take seeing a `student*` page used on an instructor route, but this makes sense because we're looking at a student's instance question. I looked at `studentGeneratedFilesQuestion` and `instructorGeneratedFilesQuestion` and they're virtually identical; I think we could merge them into a single `generatedFilesQuestion` page in a separate PR. We could unconditionally look at `res.locals.question.id`, just like `clientFilesQuestion`.

Reply:
Actually, that might need a little more consideration so that students can't access generated files from variants that they don't own...

Reply:
I opened #5500 with this refactor.